test/test.js

```diff
@@ -138,6 +138,18 @@ describe( 'rollup-plugin-babel', function () {
 		});
 	});
 
+	it( 'allows transform-runtime to be used instead of bundled helpers, but throws when CommonJS is used', () => {
+		return bundle(
+			'samples/runtime-helpers-commonjs/main.js',
+			{ runtimeHelpers: true }
+		).then(() => {
+			assert.ok(false);
+		}).catch((error) => {
+			// It looks like your Babel configuration specifies a module transformer.
+			assert.ok( ~error.message.indexOf( 'It looks like your Babel configuration specifies a module transformer. Please disable it.' ) );
```

Comment:
While we are at it, maybe it would be helpful for people to give some more info in this error? Smth like:
```js
It looks like your Babel configuration specifies a module transformer. Please disable it.

Most commonly it's a CommonJS transform added by @babel/preset-env or @babel/preset-2015 - in such case you should disable it by adding `modules: false` option to that preset (described in more detail here - https://github.com/rollup/rollup-plugin-babel#modules ).
```

Reply:
I don't think we have to mention es2015 anymore in babel v7. Was deprecated such a long time ago. But clearly it's better to be a little more helpful.